### DISCUSSION TITLE
Removing red exclamation mark circle from icon

### DISCUSSION THREAD

**Discussion by Mamunson** [twbs/bootstrap#38721] [Q&A] [answered]
Hello everyone,

I got an exclamation mark in the middle of the flag which I'm trying to remove, this is the code:

`  <a class="c-btn c-btn--flush" href="https://en.mypage.com" >
   <span class="fa-layers fa-fw fa-3x">
      <i class="fas fa-circle"></i>
      <i class="fab fi-us fis"
   data-fa-transform="shrink-8" ></i>  </span>  
  </a>` 

I included this link in my header.php file: [   <link
  rel="stylesheet"
  href="https://cdn.jsdelivr.net/gh/lipis/flag-icons@6.6.6/css/flag-icons.min.css"
/>
](url)

I've attached a screenshot. 
![screenshot-icons](https://github.com/twbs/bootstrap/assets/88907193/7dd69900-18a3-41ac-8480-a05025d9d193)
I'll be grateful for your help.

**Comment by ffoodd** [comment, marked as answer]
This doesn't look like Bootstrap code.

Reading your markup, you have two Font Awesome icons in your link, I guess you need to remove the first one.

**Reply by Mamunson** [reply]
Many thanks for your help.
